app/src/main/kotlin/com/wire/android/ui/conversation/mention/model/Mention.kt

```diff
@@ -0,0 +1,16 @@
+package com.wire.android.ui.conversation.mention.model
+
+import com.wire.android.ui.conversation.all.model.Conversation
+
+data class Mention(
+    val mentionInfo: MentionInfo,
+    val conversation: Conversation
+)
+
+data class MentionInfo(val mentionMessage: MentionMessage)
+
+data class MentionMessage(val message: String) {
+    fun toQuote(): String {
+        return "\"$message\""
+    }
```

Comment:
I'm not sure if I prefer tripple-quoted raw strings or just escaping the quotes. 
In any case, I'd suggest to just use an `=` instead of opening the function block and `return` statement:

```suggestion
    fun toQuote() = "\"$message"\""
```

```suggestion
    fun toQuote() = """"$message""""
```

Raw strings are pretty awesome, but don't quite make the problem any better when one is trying to avoid scaping quotes 😆